docs/compute-element/hosted-ce.md

```diff
@@ -4,6 +4,8 @@ DateReviewed: 2021-01-28
 Requesting an OSG Hosted CE
 ===========================
 
+!!! info "For assistance starting a new OSG integration, please reach out to support@osg-htc.org"
+
```

Comment:
Check out some of the other admonitions that we use: you'll want a summary title + explanatory body. Maybe something like the following if it reads ok to you:

```suggestion
!!! info "Schedule a consultation"
    To ensure that we can meet your research computing goals needs, please contact us at <support@osg-htc.org> to schedule a consultation.
```